test/Microsoft.AspNetCore.Mvc.DataAnnotations.Test/project.json

```diff
@@ -16,6 +16,7 @@
     "Microsoft.NETCore.Platforms": "1.0.1-*",
     "xunit.runner.aspnet": "2.0.0-aspnet-*"
```

Comment:
Wouldn't you need this for coreclr? I haven't switched the dnxcore tests to use `dotnet test` as yet.